### PR TITLE
Cleanup and bump appVersions

### PR DESCRIPTION
**Pull request by onedr0p**
#### Special notes for your reviewer:

Bumping versions to their latest releases and updating the chart.yaml's to include the actual app versions (not docker image version)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/zwave2mqtt/values.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ strategyType: Recreate
 
 image:
   repository: robertslando/zwave2mqtt
-  tag: 3.0.1
+  tag: 4.0.3
```
Comment (billimek) [inline charts/zwave2mqtt/values.yaml:10]: This version caused some issues with excessive logging. I suggest not updating this chart just yet.

Comment (onedr0p) [reply charts/zwave2mqtt/values.yaml:10]: This seems reasonable to me... 
image

Comment (onedr0p) [reply charts/zwave2mqtt/values.yaml:10]: Any chance your log file wasn't pruned after upgrading?

Comment (billimek) [reply charts/zwave2mqtt/values.yaml:10]: It does get pruned.  This is what I see after upgrading to 4.0.3.  It seems to get into an endless logging loop after changing the power state of a zwave switch.  I'll  log an issue over there.  Maybe it's a problem that only affects certain setups:

image

That's approx 30KB/sec log growth rate.

Comment (onedr0p) [reply charts/zwave2mqtt/values.yaml:10]: Gotcha, I'll bump this back down to 3.4.1

Comment (billimek) [reply charts/zwave2mqtt/values.yaml:10]: With that said, I'm good either way with the version you have set - it's easy to override.  I did raise https://github.com/OpenZWave/Zwave2Mqtt/issues/697 for the issue.

Comment (onedr0p) [reply charts/zwave2mqtt/values.yaml:10]: Not sure about the lint issue, maybe because this is so large, but we can get it merged as is.

**Pull request merged**